_config.yml

```diff
@@ -39,6 +39,11 @@ markdown: kramdown
 sass:
   style: compressed
 
+scripts:
+  - /assets/uswds/js/uswds.min.js
```

Comment:
This is necessary because the USWDS script from the base `uswds-jekyll` theme is only included by default if the site doesn't define its own `scripts` setting:

https://github.com/18F/uswds-jekyll/blob/472991d/_includes/scripts.html#L4-L7

The approach here is the same as taken in the handbook:

https://github.com/18F/handbook/blob/3875342/_config.yml#L26

It does raise a question whether this is a sensible default behavior for the theme, or if instead `uswds.min.js` should always be pushed into the scripts list regardless of whether the site defines its own scripts. I suppose it's a question of whether to optimize for scripts configuration as a _substitute_ to the default, or as an _addition_ to.

Reply:
Thanks! I don't have strong opinions either way. Pretty much I just do what ever the handbook or before-you-ship guide does. My end of FY20 goal is to sync up with the USWDS 2.0 jekyll template, abstract away all the stuff those handbooks have done into something reusable across most guides.